### PR TITLE
Add documents on "HSF Affiliated Projects and Software" and related Guidelines

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Document as finalised by Pere and I with community input, and agreed within the SG.

I started a new folder for this, which should become the one for the top-level menu "Projects & Support" on the homepage.

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
This is now ready. The only things missing are the links for the AG (to be added once available) and the documents on the guidelines, which will need to be finalised soon and put online as well.

I'm not suggesting that anyone reviews at this time on a Friday. Let's finalise next week.

**Comment by eduardo-rodrigues** [comment]
Kind ping for 1-2 of you to review this MR :-).

File: projects/affiliated.md
Comment (hegner) [inline projects/affiliated.md:12]: wouldn't use the word remit and just talk about goals.

File: projects/affiliated.md
Comment (hegner) [inline projects/affiliated.md:37]: why "affiliated" in brackets?

File: projects/affiliated.md
Comment (hegner) [inline projects/affiliated.md:39]: No quotation marks - that are genuine reviews, no?

**Review by hegner — CHANGES_REQUESTED** [review]
Just minor points. Thank you!

File: projects/affiliated.md
Comment (eduardo-rodrigues) [reply projects/affiliated.md:37]: Oh, that's a left-over from the times we made a distinction. Thanks for spotting it. Removing the parentheses.

File: projects/affiliated.md
Comment (eduardo-rodrigues) [reply projects/affiliated.md:12]: OK.

Comment (eduardo-rodrigues) [inline projects/affiliated.md:12]:
```suggestion
*HSF Affiliated Projects and Software* are community projects or software packages with person-power, and possibly dedicated funding, which connect strongly to the HSF and align with [its goals]({{ site.baseurl }}/organization/goals.html). They benefit from their inclusion in, or association with, the HSF through access to the community for wide visibility and easier promotion through the network. Affiliation makes it clear that the HSF has no control over the endeavours/products, hence no responsibility for evolution or maintenance.
```

File: projects/affiliated.md
Comment (eduardo-rodrigues) [inline projects/affiliated.md:39]:
```suggestion
## HSF Reviews
```

**Comment by eduardo-rodrigues** [comment]
Thanks. Approve when happy. I will wait for a second approval given the importance of the document.

**Comment by hegner** [comment]
@eduardo-rodrigues - would have suggested the same - another pair of eyes would be good. I am fine with the current state

**Review by hegner — APPROVED** [review]

**Review by graeme-a-stewart — CHANGES_REQUESTED** [review]
Thank you @eduardo-rodrigues - I made some suggestions here.

Before we accept it seems we need:

- The guidelines document
- Updated `navbar.ext` that links to these documents

**Comment by graeme-a-stewart** [comment]
> This is now ready. The only things missing are the links for the AG (to be added once available) 

It will be `/organization/advisory-group.html` - see #1597.

**Comment by graeme-a-stewart** [comment]
Sorry it took a while to review @eduardo-rodrigues - JuliaHEP was really full-on last week!

File: projects/affiliated.md
Comment (eduardo-rodrigues) [inline projects/affiliated.md:18]:
```suggestion
Anyone is welcome to engage with the HSF in general and the
```

File: projects/affiliated.md
Comment (eduardo-rodrigues) [inline projects/affiliated.md:6]:
```suggestion
```

Comment (eduardo-rodrigues) [reply projects/affiliated.md]: OK, I did not realise how the templates work.

File: projects/affiliated.md
Comment (eduardo-rodrigues) [reply projects/affiliated.md]: OK

File: projects/affiliated.md
Comment (eduardo-rodrigues) [reply projects/affiliated.md]: OK

File: projects/affiliated.md
Comment (eduardo-rodrigues) [reply projects/affiliated.md:11]: You mean "HSF has no control" -> "HSF has no direct control"? Sure.

File: projects/affiliated.md
Comment (graeme-a-stewart) [reply projects/affiliated.md:27]: Yeah, I agree about that. Let's try to get both done together.

File: projects/affiliated.md
Comment (eduardo-rodrigues) [inline projects/affiliated.md:11]:
```suggestion
*HSF Affiliated Projects and Software* are community projects or software packages with person-power, and possibly dedicated funding, which connect strongly to the HSF and align with [its goals]({{ site.baseurl }}/organization/goals.html). They benefit from their inclusion in, or association with, the HSF through access to the community for wide visibility and easier promotion through the network. Affiliation makes it clear that the HSF has no direct control over the endeavours/products, hence no responsibility for evolution or maintenance.
```

File: projects/affiliated.md
Comment (eduardo-rodrigues) [reply projects/affiliated.md:7]: "Introduction" is just fine, yes :-).

File: projects/affiliated.md
Comment (eduardo-rodrigues) [inline projects/affiliated.md:7]:
```suggestion
## Introduction
```

File: projects/affiliated.md
Comment (eduardo-rodrigues) [reply projects/affiliated.md]: Good idea. Added.

**Comment by eduardo-rodrigues** [comment]
> > This is now ready. The only things missing are the links for the AG (to be added once available)
> 
> It will be `/organization/advisory-group.html` - see #1597.

Link added.

File: projects/affiliated.md
Comment (eduardo-rodrigues) [reply projects/affiliated.md:7]: This is a nice idea and I added your sentence verbatim.

File: projects/affiliated.md
Comment (eduardo-rodrigues) [reply projects/affiliated.md:32]: Added in the most recent commit.

File: projects/affiliated.md
Comment (eduardo-rodrigues) [reply projects/affiliated.md:42]: Added.

**Comment by eduardo-rodrigues** [comment]
Hi all. I believe I have now done everything expected :-). I hope I got all (cross-)links correct but good that a couple of us check ...

Related to this update, we need to discuss what exactly to do with the tab highlighted below:
image

For now I moved all 3 items to below the line, having the 2 new documents here above the line. This makes sense to me. This being said, the "Member projects" and "How to join as a project" need to be revamped or deleted. Shall we do that in a quick follow-up?

**Review by graeme-a-stewart — APPROVED** [review]

**Comment by graeme-a-stewart** [comment]
Thanks @eduardo-rodrigues - this is a lot of work!

I pushed a [commit](https://github.com/HSF/hsf.github.io/pull/1595/commits/c81fa5a7e2d22cf7c155e2a57560df1635c13373) that:

- Fixes navbar links
- Shortens the overlong titles
- Fixes a typo

If you are good with that change I think we can merge.

**Comment by eduardo-rodrigues** [comment]
Many thanks for the careful checks! I see 2 approvals, merging then :-).

**Pull request merged**
